### DISCUSSION TITLE
case insensitive search

### DISCUSSION THREAD

**Discussion by pyramation** [BowlingX/ra-postgraphile#87] [General]
I'm using search/filter a bit and seems that by default it's all case-sensitive... is it a simple switch or does this require a PR?

**Comment by BowlingX** [comment]
You can use the `filterToQuery` prop, see https://marmelab.com/react-admin/Inputs.html#usage-11.

```ts
const filterToQuery = (searchText: string) => ({
  yourAttribute: {
    value: searchText,
    operator: 'startsWithInsensitive'
  },
})
```
If you have the filter plugin enabled (https://github.com/graphile-contrib/postgraphile-plugin-connection-filter), you can specify the search operator in `operator`.

**Reply by pyramation** [reply]
is there a similar feature for `SearchInput`?

```
    <SearchInput
      placeholder="Search Name"
      source="name"
      filterToQuery={(searchText) => ({
        name: {
          value: searchText,
          operator: 'likeInsensitive'
        }
      })}
      alwaysOn
    />

```

It doesn't seem to be working for me. It does work for the `ReferenceInput`, but currently seemingly not `SearchInput`
